rm-datasource/src/test/java/io/seata/rm/datasource/sql/struct/TableMetaTest.java

```diff
@@ -318,6 +288,7 @@ public ResultSet executeQuery(MockStatementBase statement, String s) throws SQLE
                 new Object[] {"", "", "t1", "name3", Types.VARCHAR, "VARCHAR", 64, 0, 10, 0, "", "", 0, 0, 64, 5, "YES", "NO"},
                 new Object[] {"", "", "t1", "id_card", Types.DECIMAL, "DECIMAL", 20, 0, 10, 1, "", "", 0, 0, 64, 4, "NO", "YES"}
             };
+        mockDriver.setColumnMetas(columnMetas);
```

Comment:
Here is an array of references, need to be set each time?

Reply:
Yes, If you want to mock change the table struct